src/site/content/en/blog/file-system-access/index.md

```diff
@@ -210,83 +215,147 @@ async function writeURLToFile(fileHandle, url) {
+    console.log(entry.kind, entry.name);
   }
 });
 ```
 
+### Creating or accessing files and folders in a directory
+
+From a directory, you can create or access files and folders using the
+[`getFileHandle()`][getfilehandle] or respectively the [`getDirectoryHandle()`][getdirectoryhandle]
+method. By passing in an optional `options` object with a key of `create` and a boolean value of
+`true` or `false`, you can determine if a new file or folder should be created if it doesn't exist.
+
+```js
+// In an existing directory, create a new directory named "My Documents".
+const newDirectoryHandle = await existingDirectoryHandle.getDirectoryHandle('My Documents', {
+  create: true,
+});
+// In this new directory, create a file named "My Notes.txt".
+const newFileHandle = await newDirectoryHandle.getFileHandle('My Notes.txt', { create: true });
+```
+
+### Resolving the path of an item in a directory
+
+When working with files or folders in a directory, it can be useful to resolve the path of the item
+in question. This can be done with the aptly named [`resolve()`][resolve] method. For resolving, the
+item can be a direct or indirect child of the directory.
+
+```js
+// Resolve the path of the previously created file called "My Notes.txt".
+const path = await newDirectoryHandle.resolve(newFileHandle);
+// `path` is now ["My Documents", "My Notes.txt"]
+```
+
+### Deleting files and folders in a directory
+
+If you have obtained access to a directory, you can delete the contained files and folders with the
+[`removeEntry()`][removeentry] method, for folders optionally recursively including all subfolders
+and the therein contained files.
+
+```js
+// Delete a file.
+await directoryHandle.removeEntry('Abandoned Masterplan.txt');
+// Recursively delete a folder.
+await directoryHandle.removeEntry('Old Stuff', { recursive: true });
+```
+
+### Accessing the origin-private file system
+
+The origin-private file system is a storage endpoint that, as the name suggests, is private to the
+origin of the page. While browsers will typically implement this by persisting the contents of this
+origin-private file system to disk somewhere, it is _not_ intended that the contents be easily user
+accessible. Similarly, there is _no_ expectation that files or directories with names matching the
+names of children of the origin-private file system exist. You can operate as usual on the
+origin-private file system once you have access to the root `FileSystemDirectoryHandle`.
+
+```js
+const root = await navigator.storage.getDirectory();
+// Create a new file handle.
+const fileHandle = await root.getFileHandle('Untitled.txt', { create: true });
+// Create a new directory handle.
+const dirHandle = await root.getDirectoryHandle('New Folder', { create: true });
+// Recursively remove a directory.
+await root.removeEntry('Old Stuff', { recursive: true });
+```
+
+## Polyfilling
+
+It is not possible to completely polyfill the File System Access API methods.
```

Comment:
This implies that I can polyfill its properties. Is that true?

Reply:
Not the properties, but the behavior. Mostly. I've outlined in the bullets what's possible and what not.